paddle/fluid/operators/mkldnn/softmax_mkldnn_op.cc

```diff
@@ -145,27 +157,16 @@ class SoftmaxMKLDNNGradKernel : public paddle::framework::OpKernel<T> {
                           "Operator DNNL SoftmaxGrad must use CPUPlace"));
     auto& dev_ctx = ctx.template device_context<MKLDNNDeviceContext>();
     const Tensor* output = ctx.Input<Tensor>("Out");
-    auto* dout = ctx.template Input<Tensor>(framework::GradVarName("Out"));
-    auto* dx =
+    auto* out_grad = ctx.template Input<Tensor>(framework::GradVarName("Out"));
+    auto* in_x_grad =
         ctx.template Output<framework::Tensor>(framework::GradVarName("X"));
```

Comment:
```suggestion
        ctx.template Output<Tensor>(framework::GradVarName("X"));
```
Please, stay consistent with the usage of namespaces

Reply:
ok